internal/kafka/internal/presenters/kafka.go

```diff
@@ -31,28 +33,48 @@ func ConvertKafkaRequest(kafkaRequestPayload public.KafkaRequestPayload, dbKafka
 }
 
 // PresentKafkaRequest - create KafkaRequest in an appropriate format ready to be returned by the API
-func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest) public.KafkaRequest {
+func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest, config *config.KafkaConfig) public.KafkaRequest {
 	reference := PresentReference(kafkaRequest.ID, kafkaRequest)
-
+	var ingressThroughputPerSec, egressThroughputPerSec, maxDataRetentionPeriod string
+	var totalMaxConnections, maxPartitions, maxConnectionAttemptsPerSec int
+	if config != nil {
+		kafkaConfig, err := config.GetKafkaInstanceSize(kafkaRequest.InstanceType, kafkaRequest.SizeId)
+		if err != nil {
+			glog.Warning(err)
+		} else {
+			ingressThroughputPerSec = kafkaConfig.IngressThroughputPerSec
+			egressThroughputPerSec = kafkaConfig.EgressThroughputPerSec
+			totalMaxConnections = kafkaConfig.TotalMaxConnections
+			maxPartitions = kafkaConfig.MaxPartitions
+			maxDataRetentionPeriod = kafkaConfig.MaxDataRetentionPeriod
+			maxConnectionAttemptsPerSec = kafkaConfig.MaxConnectionAttemptsPerSec
+		}
```

Comment:
Not strictly related to the code per-se, but If the configuration has been changed since when the cluster has been created, returned infos will be wrong

Reply:
@ziccardi Yeah, but how can we bet the latest values here? Shouldn't the fleet-manager pod be restarted once the configuration is changed?

Reply:
I don't think the kafka instance will be destroyed if we change the configuration. This is something that can happen. With the current implementation, there is no way to return the actual infos (unless I miss something). My comment was just to document a known, potential, issue.

Reply:
Yeah, I've tested with this and it does happen. Once you create the Kafka with say x1 and standard and then you remove the x1 from the standard array but it's still there in the kafka_requests table, the API will not return these parameters. How should we deal with this? @pb82, @JameelB

Reply:
that's a really good point. That doesn't just affect this information but the way capacity is calculated as well. 

We could check that there's no Kafka with the specific size/instance type before it can be removed from the config. But even then, while the change is deployed someone else could create a Kafka of that size/instance type. There isn't a way for us to disable creation of certain sizes atm without removing them from the config.

@pb82 we may need to go back to that conversation of adding these to the kafka_requests table :sweat_smile:

Reply:
After recent discussions on this, for now we've agreed that once an instance type/size has been added to the configuration, it should not be removed. If needed, we can scope out this work in a separate follow on PR/task. 

Is this ok with you @ziccardi @pb82?